addons/l10n_at/data/account_fiscal_position_template.xml

```diff
@@ -16,6 +23,13 @@
             <field name="country_group_id" ref="base.europe" />
             <field name="chart_template_id" ref="l10n_at_chart_template"/>
         </record>
+        <record id="fiscal_position_template_eu_wo_uid" model="account.fiscal.position.template">
+            <field name="name">Europäische Union (ohne UID)</field>
```

Comment:
Haha, ask @william-andre, so yeah, it is all about automatically detect the fiscal position based on the VAT. If you are from the countries inside the EU and do not have a VAT (which might happen sometimes), we do not want to apply the Non EU FP. Even worse for Austrian countries having a VAT which would now be detected as FP EU instead of National.

Besides these I have really a lot of FP in place for dealing with more complicated stuff, but for Standard I think this should help to have a basic set without running into wrong detections.

Reply:
Ahh, and besides this, it exists for B2B only, as OSS only applies to B2C and therefore we can not apply the related OSS FP as they would not be allowed to deduct the tax. They get billed (as long as the company does not hit the limit to create a foreign EU VAT) the local tax. This is why there is no tax mapping, but it is a totally different use case and therefore also a different FP.

Reply:
Oooh ok, thanks for the explanation :)